langroid/agent/chat_agent.py

```diff
@@ -322,9 +322,9 @@ def llm_response(
         if not self.llm_can_respond(message):
             return None
 
-        assert (
-            message is not None or len(self.message_history) == 0
-        ), "message can be None only if message_history is empty, i.e. at start."
+        assert message is not None or len(self.message_history) == 0, (
+            "message can be None only if message_history is empty, i.e. at" " start."
```

Comment:
there are many such changes interspersed among the meaningful changes due to the lint diff. If VSC is changing things, something needs to be changed there, or immediately after VSC changes things, inspect the file and undo those in a different editor, or some other idea, not sure. In particular this change is silly: `".... at" " start.".` I suggest inspecting the changes and editing manually.

Reply:
I will be making these changes manually on top of your PR for now.